diff_drive_controller/src/diff_drive_controller.cpp

```diff
@@ -241,6 +245,8 @@ namespace diff_drive_controller{
 
     setOdomPubFields(root_nh, controller_nh);
 
+    cmd_vel_pub_.reset(new realtime_tools::RealtimePublisher<geometry_msgs::TwistStamped>(controller_nh, "cmd_vel_out", 100));
```

Comment:
Yeah, that's probably better. Allocating it would create a topic that never gets published on.